### ISSUE TITLE
Name property is not defined for 'Mute', 'CC' and 'Settings' button.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1380]
**Actual Result:**
Name property is not defined for 'Mute', 'CC' and 'Settings' button.

**Note:**
Incorrect time information is defined for 'seek slider'.

**Expected Result:**
Name property should be defined for 'Mute', 'CC' and 'Settings' button.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Start Narrator by pressing 'Ctrl + Win + Enter' key.
3. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
4. Navigate to 'Show a video attachment from YouTube' button and activate it.
5. YouTube video will appear at the bottom of the screen.
6. Navigate to mentioned control and observe if name property is defined of not.

**User Impact:**
If name property is not defined for any control then the user who depends on screen reader will not able to understand the purpose of that control.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**MAS Reference:**
[MAS 4.1.2 - Name, Role, Value (40B)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
[MAS4.1.2_NameProperty.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2599114/MAS4.1.2_NameProperty.zip)

**Comment by compulim** [comment]
This is not a bug from Web Chat. The video content is an IFRAME owned by YouTube.

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with Accessibility team as an external issue.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
